tools/ci_build/github/azure-pipelines/nuget/templates/gpu.yml

```diff
@@ -110,6 +169,40 @@ jobs:
      copy $(Build.BinariesDirectory)\nuget-artifact\*.nupkg $(Build.ArtifactStagingDirectory)
     displayName: 'Bundle Native NuGet and other binaries'
 
+  - script: |
+     pushd $(Build.BinariesDirectory)\nuget-artifact-dml
+     dir
+     powershell -Command "Invoke-WebRequest http://stahlworks.com/dev/unzip.exe -OutFile unzip.exe"
+     powershell -Command "Invoke-WebRequest http://stahlworks.com/dev/zip.exe -OutFile zip.exe"
+     set PATH=%CD%;%PATH%
+     FOR /R %%i IN (*.nupkg) do ( 
+         rename %%~ni.nupkg %%~ni.zip
+         unzip %%~ni.zip -d %%~ni
+         del /Q %%~ni.zip
+
+         unzip win-dml-x86.zip -d win-x86
+         mkdir %%~ni\runtimes\win-x86
+         mkdir %%~ni\runtimes\win-x86\native
+
+         move win-x86\runtimes\win-x86\native\onnxruntime.dll %%~ni\runtimes\win-x86\native\onnxruntime.dll
+         move win-x86\runtimes\win-x86\native\onnxruntime.lib %%~ni\runtimes\win-x86\native\onnxruntime.lib
+         move win-x86\runtimes\win-x86\native\onnxruntime.pdb %%~ni\runtimes\win-x86\native\onnxruntime.pdb
+
+         move win-x86\runtimes\win-x86\native\windows.ai.machinelearning.dll %%~ni\runtimes\win-x86\native\windows.ai.machinelearning.dll
+         move win-x86\runtimes\win-x86\native\windows.ai.machinelearning.lib %%~ni\runtimes\win-x86\native\windows.ai.machinelearning.lib
+         move win-x86\runtimes\win-x86\native\windows.ai.machinelearning.pdb %%~ni\runtimes\win-x86\native\windows.ai.machinelearning.pdb
+
+         move win-x86\runtimes\win-x86\native\directml.dll %%~ni\runtimes\win-x86\native\directml.dll
+
+         pushd %%~ni
+         zip -r ..\%%~ni.zip .
+         popd
+         move %%~ni.zip %%~ni.nupkg
+     )
+     popd
```

Comment:
Powershell can do zipping/unzipping natively (using Compress-Archive and Expand-Archive). Perhaps that would be a better option, rather than relying on third-party tools?

Reply:
Oh, wait, I see that the zip/unzip stuff was copied from above. Feel free to disregard this then. :)

---
In reply to: [386732399](https://github.com/microsoft/onnxruntime/pull/3120#discussion_r386732399) [](ancestors = 386732399)